### PR TITLE
Breadcrumbs styling first draft - based on 3.5.x

### PR DESCRIPTION
**Pull request by mmexvr**
**Describe pull-request**</br>
_Intial styling for breadcrumbs_
- Font size
- background-color
- Padding
- Active item

**Solving issue**</br>
Add which issue this pull-request solves by adding # plus the number of the issue (for example #123)
Fixes breadcrumbs in : #11 
Fixed task: #26 

**Additional context**<br/>
Based on ux-pin

### CODE REVIEW

**Comment by mmexvr** [comment]
Conflict solved

**Comment by mmexvr** [comment]
Side note: Need to add breadcrumbs to storybook when this is merge

File: src/styles/components/_breadcrumbs.scss [hunk 1]
```diff
@@ -0,0 +1,30 @@
+@import '../variables';
+
+.breadcrumb {
+  font-size: 16px;
+  background-color: #F5F5F5;
```
Comment (mmexvr) [inline src/styles/components/_breadcrumbs.scss]: Fix color

**Review by helloimela — APPROVED** [review]
Looks good 👍

**Pull request merged**